recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
and I think @ghorn is the debian Ipopt maintainer these days, any plan to package the ampl solver library there (under whatever name) so debian's ipopt will have an executable?